### PR TITLE
add Go example for submitting to bridge server

### PR DESCRIPTION
**Pull request by alien35**

### CODE REVIEW

File: guides/anchor/2-bridge-server.md [hunk 1]
```diff
@@ -149,6 +149,29 @@ public class PaymentRequest() {
...
+request_url := "http://localhost:8006/payment"
+form := url.Values{
+  "amount":       {"1"},
+  "asset_code":   {"USD"},
+  "asset_issuer": {"GAIUIQNMSXTTR4TGZETSQCGBTIF32G2L5P4AML4LFTMTHKM44UHIN6XQ"},
+  "destination":  {"GCFXHS4GXL6BVUCXBWXGTITROWLVYXQKQLF4YH5O5JT3YZXCYPAFBJZB"},
+  "source":       {"SAV75E2NK7Q5JZZLBBBNUPCIAKABN64HNHMDLD62SZWM6EBJ4R7CUNTZ"},
+}
+body := bytes.NewBufferString(form.Encode())
+resp, err := http.Post(request_url, "application/x-www-form-urlencoded", body)
```
Comment (bartekn) [inline guides/anchor/2-bridge-server.md]: [`PostForm`](https://golang.org/pkg/net/http/#Client.PostForm) would be easier to use.

File: guides/anchor/2-bridge-server.md [hunk 2]
```diff
@@ -149,6 +149,29 @@ public class PaymentRequest() {
...
+if err != nil {
+  panic(err)
+}
+defer resp.Body.Close()
```
Comment (bartekn) [inline guides/anchor/2-bridge-server.md]: Can we also print a response, just like in other examples?

**Comment by tomerweller** [comment]
Closing due to inactivity. @alien35 please reopen with suggested changes.

**Comment by tomerweller** [comment]
Closing due to inactivity. @alien35 please reopen with suggested changes.

**Pull request closed without merging**